src/twisted/test/proto_helpers.py

```diff
@@ -925,3 +927,64 @@ def _check():
     lc = task.LoopingCall(_check)
     lc.clock = reactor
     return lc.start(0.01, now=True)
+
+
+
+@implementer(ILogObserver)
+class EventLoggingObserver(Sequence):
+    """
+    L{ILogObserver} That stores its events in a list for later inspection.
+    This class is similar to L{LimitedHistoryLogObserver} save that the
+    internal buffer is public and intended for external inspection.  The
+    observer implements the sequence protocol to ease iteration of the events.
+
+    @ivar _events: The events captured by this observer
+    @type _events: C{list}
```

Comment:
This should be `L{list}` as specified in the coding standard: http://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html#docstrings

Reply:
Yep, will do, sorry for the miss.